dlib/external/pybind11/tools/pybind11Tools.cmake

```diff
@@ -145,7 +152,6 @@ function(pybind11_add_module target_name)
   # namespace; also turning it on for a pybind module compilation here avoids
   # potential warnings or issues from having mixed hidden/non-hidden types.
   set_target_properties(${target_name} PROPERTIES CXX_VISIBILITY_PRESET "hidden")
-  set_target_properties(${target_name} PROPERTIES CUDA_VISIBILITY_PRESET "hidden")
```

Comment:
Same here, this was added in 3de8fa3688b9fdf21b63cba0bc53dc322f196126 and we should keep it.